witness/src/Dscp/DB/Rocks/Real/Functions.hs

```diff
@@ -37,7 +46,9 @@ closeRocksDB :: MonadIO m => DB -> m ()
 closeRocksDB = Rocks.close . rocksDB
 
 openNodeDB :: MonadIO m => RocksDBParams -> m RocksDB
-openNodeDB RocksDBParams{..} = RocksDB <$> openRocksDB rdpPath
+openNodeDB RocksDBParams{..} = liftIO $ do
+    whenM (doesDirectoryExist rdpPath) $ removeDirectoryRecursive rdpPath
```

Comment:
what? you remove the database directory every time you open the database or what? if so, this seems mildly wrong...

Reply:
I assume that here you intended to also check the `rdpClean` flag (now it isn't used at all).

Reply:
Yes, sure, I wasn't yet looking on what I coded :p

Reply:
fixed